packages/nextjs/components/ActivitiesItem.tsx

```diff
@@ -0,0 +1,225 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { ethers } from "ethers";
+import { isAddress } from "ethers/lib/utils";
+import Blockies from "react-blockies";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { hardhat } from "wagmi/chains";
+import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
+import { getBlockExplorerAddressLink, getTargetNetwork } from "~~/utils/scaffold-eth";
+
+/**
+ * Displays an address (or ENS) with a Blockie image and option to copy address.
+ */
+
+const Address = ({ address }: { address: string }) => {
+  const [addressCopied, setAddressCopied] = useState(false);
+  const displayAddress = address?.slice(0, 5) + "..." + address?.slice(-4);
+  const blockExplorerAddressLink = getBlockExplorerAddressLink(getTargetNetwork(), address);
+
+  return (
+    <div className="flex">
+      {getTargetNetwork().id === hardhat.id ? (
+        <span className={`ml-1.5 text-sm font-normal`}>
+          <Link href={blockExplorerAddressLink}>{displayAddress}</Link>
+        </span>
+      ) : (
+        <a
+          className={`ml-1.5 text-sm font-normal`}
+          target="_blank"
+          href={blockExplorerAddressLink}
+          rel="noopener noreferrer"
+        >
+          {displayAddress}
+        </a>
+      )}
+      {addressCopied ? (
+        <CheckCircleIcon
+          className="ml-1.5 text-xl font-normal text-sky-600 h-5 w-5 cursor-pointer"
+          aria-hidden="true"
+        />
+      ) : (
+        <CopyToClipboard
+          text={address}
+          onCopy={() => {
+            setAddressCopied(true);
+            setTimeout(() => {
+              setAddressCopied(false);
+            }, 800);
+          }}
+        >
+          <DocumentDuplicateIcon
+            className="ml-1.5 text-xl font-normal text-sky-600 h-5 w-5 cursor-pointer"
+            aria-hidden="true"
+          />
+        </CopyToClipboard>
+      )}
+    </div>
+  );
+};
+
+export type TActivityItemProps = {
+  address: string;
+  amount: number;
+  landedOn: string;
+};
+
+export const ActivitiesItems = ({ address, amount, landedOn }: TActivityItemProps) => {
```

Comment:
I think it should be `ActivityItem`

Reply:
i have renamed the component  at [4bd648d](https://github.com/scaffold-eth/se-2-challenges/pull/48/commits/4bd648d28147d24104d096674e2da05163845518)